### ISSUE TITLE
Some Possible Improvements?

### ISSUE THREAD

**Issue by SeiferTim** [AxolStudio/STLGameDevRoundUp#11]
How do we get each issue out to the people who are most interested in seeing it?

I guess I can setup an RSS feed (do people use those anymore?) 
I could do something like a mailing list - people submit their email and we send an email when a new issue goes out (but we have to be careful about spam and let them remove themselves from the list.. maybe use 3rd party service)
I think we could do something where people can subscribe to push-alert notifications, too?

What do people think?

**Comment by jessechounard** [comment]
I receive lots of weekly/biweekly tech newsletters via mailing lists. I think that's a fantastic way to handle this. If you would like me to look into seeing what services they run them on, I'm happy to.

**Comment by SeiferTim** [comment]
> I receive lots of weekly/biweekly tech newsletters via mailing lists. I think that's a fantastic way to handle this. If you would like me to look into seeing what services they run them on, I'm happy to.

I made this to intentionally avoid any kind of emails. I've messed with stuff like MailChimp etc in the past and they end up being a pain. ...and most of the emails end up in spam and/or never opened.

I also wanted this to be sort of 'decentralized' allowing anyone to contribute - and having a list of peoples' emails sitting around that is public doesn't feel right.

I feel like the current state of the internet is no longer set up for people to 'passively' get alerted to when something they care about gets an update.

RSS feeds used to be a big thing - now the only way I know when a site I care about has an update is if I happen to catch it on Twitter 🤮 

Anyway, I think the ideal solution here is for people to be able to opt-in to different ways to get notified that a new roundup is out - without having to submit their personal information to 'us' to have to keep track of and 'protect' that information.

I don't know what that is other than 'follow on twitter'...

**Comment by DrewUniverse** [comment]
> I feel like the current state of the internet is no longer set up for people to 'passively' get alerted to when something they care about gets an update.

This is what happened as we migrated into consolidated social media hubs as a society - both for better and worse. Maintaining our own personal list of things we want to keep track of is more daunting than merely clicking "Subscribe" on 10-100 subreddits, liking/following FB pages and groups, or following people on Twitter. Consolidation is helpful, but it also trained us to let random companies curate the content we like to see. IMO it'd be great if there were more self-sustaining feeds out there, but getting the general public to adopt that seems like its own nightmare. IDK what the solution is, but I agree it's a challenge we're faced with.

No matter what is chosen, it can be good and/or bad. A simple Twitter account sounds good, but there's also the general instability of that service post-Elon. Personally I think this sort of thing just belongs on the co-op website, but that takes time/resources and I'm not sure what Rob's current plans are. But even if it was added to the website, how many people will visit? 

In the end your best bet may be to just bring print-outs of the Round-Ups to events. Any of the play-and-shares, include Round-Up flyers. Holiday party? Round-Up flyers. Quick, no need to sign up on the internet, recyclable. Or instead of printing many copies, bring one single flyer that's a bit larger / more snazzy / eye-catching. Have it set like a stand-up poster or sitting on a table, people come by and read it. Change it out once a month?

These thoughts are mostly geared toward fellow gamedevs, though. As for the general gaming public in STL, that's a whole other discussion we can and should be having.
